packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```diff
@@ -204,42 +241,42 @@ export function SchedulesTable({
         <Field width="flex" name="name">
           <Text
             style={
-              item.name == null
+              schedule.name == null
                 ? { color: theme.buttonNormalDisabledText }
                 : null
             }
-            title={item.name ? item.name : ''}
+            title={schedule.name ? schedule.name : ''}
           >
-            {item.name ? item.name : 'None'}
+            {schedule.name ? schedule.name : 'None'}
           </Text>
         </Field>
         <Field width="flex" name="payee">
-          <DisplayId type="payees" id={item._payee} />
+          <DisplayId type="payees" id={schedule._payee} />
         </Field>
         <Field width="flex" name="account">
-          <DisplayId type="accounts" id={item._account} />
+          <DisplayId type="accounts" id={schedule._account} />
         </Field>
         <Field width={110} name="date">
-          {item.next_date
-            ? monthUtils.format(item.next_date, dateFormat)
+          {schedule.next_date
+            ? monthUtilFormat(schedule.next_date, dateFormat)
             : null}
         </Field>
         <Field width={120} name="status" style={{ alignItems: 'flex-start' }}>
-          <StatusBadge status={statuses.get(item.id)} />
+          <StatusBadge status={statuses.get(schedule.id)} />
         </Field>
-        <ScheduleAmountCell amount={item._amount} op={item._amountOp} />
+        <ScheduleAmountCell amount={schedule._amount} op={schedule._amountOp} />
         {!minimal && (
           <Field width={80} style={{ textAlign: 'center' }}>
-            {item._date && item._date.frequency && (
+            {schedule._date && (schedule._date as any).frequency && (
```

Comment:
Had to do as any, as defining a type to `_date` would require more additional changes, I plan to look into it when I do `EditSchedule` component as I can better understand how this _date is set when making new schedule

Reply:
Could you please add a `// eslint-disable-next-line @typescript-eslint/no-explicit-any` on the line above so the linter would pass?

Reply:
I looked into it and added a proper type 👍